app/views/departments/show.html.slim

```diff
@@ -10,14 +10,26 @@
           = render 'contacts/show', contact: @department.contact
 
 - if @department.user.any?
-  .row
-    .col-xs-12
-      h2= t_attr(:associated_users)
-  .row
-    .col-xs-12
-      ul
-        - @department.user.each do |user|
-          li= link_to user.to_label, profile_path(user.profile)
+  h2= t_attr(:associated_users)
+  ul
+    - @department.user.each do |user|
```

Comment:
This association should probably be renamed to `users`. Added a card for this at https://trello.com/c/bpZYJxVa/24-rename-departmentuser-to-users